.gitignore

```diff
@@ -70,3 +70,7 @@ typings/
 
 # next.js build output
 .next
+
+# version locks
+package-lock.json
+yarn.lock
```

Comment:
to solidify the no lock approach - please add `.npmrc` file with `package-lock=false` to avoid creating the lock file to begin with